sys/include/net/gnrc/sixlowpan/ctx.h

```diff
@@ -7,9 +7,6 @@
  */
 
 /**
- * @defgroup    net_gnrc_sixlowpan    6LoWPAN
- * @ingroup     net
- *
  * @defgroup    net_gnrc_sixlowpan_ctx    Contexts for 6LoWPAN address compression
  * @ingroup     net_gnrc_sixlowpan
  * @brief       Context buffer for stateful 6LoWPAN address compression
```

Comment:
These are not just helpers but assential to the context buffer's functionality (and thus part of the "ADT" "context buffer" ;-)).